config/default.yml

```diff
@@ -2,7 +2,7 @@ allowUnused: []
 
 devFilePatterns:
   - '{features,spec,test}/**/*'
-  - '**/*_{spec,test}.js'
+  - '**/*{spec,test}.js'
```

Comment:
What if someone has a `contest.js`.

We could do some reasonable default here like `'**/*[._-]{spec,test}.js'`.